core/core_isolate.rs

```diff
@@ -484,6 +530,25 @@ where
   callback(current_heap_limit, initial_heap_limit)
 }
 
+fn json_err(_err: ErrBox) -> Value {
```

Comment:
Actually we might want to use `JsonError` instead - then you can implement `From<ErrBox> for JsonError` and do `map_err(JsonError::from)` in `register_op_json_<>` methods. I can look into that if you want

Reply:
Feel free to push to this branch - but don't bother if it distracts you I can try this tomorrow

Reply:
I think we can avoid defining this for a first pass. PTAL at my update.